src/pages/patientView/PatientViewPageHeader.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import PatientHeader from 'pages/patientView/patientHeader/PatientHeader';
+import SampleSummaryList from 'pages/patientView/sampleHeader/SampleSummaryList';
+import { PatientViewPageStore } from 'pages/patientView/clinicalInformation/PatientViewPageStore';
+import { IGenePanelModal } from 'pages/patientView/PatientViewPage';
+import { getRemoteDataGroupStatus } from 'cbioportal-utils/src';
```

Comment:
```suggestion
import { getRemoteDataGroupStatus } from 'cbioportal-utils';
```

(caught by our CI tests)